lib/Windowing/README.MD

```diff
@@ -0,0 +1,124 @@
+# Windowing
+
+A selection of components to give you virtualisation.
+
+## How does it work?
+
+A `div` is rendered that is high enough to hold every item, it is simply calculated as follows:
+
+`listHeight = itemHeight * allWindowingIds`
+
+Each `item` has to have the same height, making all of this maths a lot easier.
+
+This `div` is wrapped in a container, with a set height, the `containerHeight`. This container is our window to look
+at the long list of items, it restricts the number of items we can see at a time though!
+
+When a scrolling event is triggered on the long list of items we retrieve a value called `scrollTop`, this represents
+the number of pixels from the top of the list that we have scrolled.
+
+With all of these values we can calculate which items can be seen through the window. Here is a quick example of the
```

Comment:
I wonder if this description helps? I wrote windowing and it confuses me a little...maybe we could simplify it by explaining key points e.g;

"How does it work? It calculates the number of components that are in view so you can render a subset of UI within a container. It does this by using a number of properties like `scrollTop`, `containerHeight` and the height of each component (this is currently fixed)."

The code explains the fact that we render a div etc...Thoughts?